micro-rdk-ffi/build.rs

```diff
@@ -18,8 +18,10 @@ pub struct Cloud {
 }
 
 fn main() {
-    #[cfg(not(feature = "test"))]
+    #[cfg(not(test))]
     {
+        println!("cargo:rustc-check-cfg=cfg(has_robot_config)");
```

Comment:
Without this, newer clippy yelled at me about `cfg(has_robot_config)` as an unknown config check.